cdap-ui/app/features/adapters/services/adapter-create-model.js

```diff
@@ -183,7 +182,7 @@ angular.module(PKG.name + '.feature.adapters')
       angular.forEach(source.properties, pruneProperties.bind(this, source));
       angular.forEach(sink.properties, pruneProperties.bind(this, sink));
 
-      for (i=0; i<trans.length; i++) {
+      for (let i=0; i<trans.length; i++) {
```

Comment:
This should have been fine on a production build since we run babel through the entire code when we do gulp build. But for development this would have been issue and since I didn't create an adapter I didn't notice this. Good catch! Fixed.